core/src/apps/solana/sign_message.py

```
@@ -0,0 +1,133 @@
+_PREAMBLE_LENGTH_LEDGER = 16 + 1 + 1 + 2
+# 1232 is the maximum length of the message with the preamble
+_MAX_MESSAGE_LENGTH_WITH_PREAMBLE = 1232
+# The maximum length of the message
+_MAX_MESSAGE_LENGTH = _MAX_MESSAGE_LENGTH_WITH_PREAMBLE - _PREAMBLE_LENGTH
+# The maximum length of the message for ledger
+_MAX_MESSAGE_LENGTH_LEDGER = _MAX_MESSAGE_LENGTH_WITH_PREAMBLE - _PREAMBLE_LENGTH_LEDGER
+
+
+@auto_keychain(__name__)
+async def sign_message(
+    ctx: wire.Context, msg: SolanaSignMessage, keychain: Keychain
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)